site/content/docs/user/pod-security-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Pod Security Policy"
+menu:
+  main:
+    parent: "user"
+    identifier: "pod-security-policy"
+    weight: 3
+---
+
+# Pod Security Policy
+
+This guide covers setting up [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
+on a kind cluster.
+
+## Enabling Pod Security Policy
+
+Pod security policy control is implemented as an optional (but recommended) admission controller. PodSecurityPolicies are enforced by enabling the admission controller, but doing so without authorizing any policies will prevent any pods from being created in the cluster.
+
+### Create Cluster
+
+Create a kind cluster with the `PodSecurityPolicy` admission plugin enabled.
+
+{{< codeFromInline lang="bash" >}}
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1beta2
+    kind: ClusterConfiguration
+    metadata:
+      name: config
+    apiServer:
+      extraArgs:
+        enable-admission-plugins: PodSecurityPolicy
+- role: worker
+EOF
+{{< /codeFromInline >}}
+
+### Install default security policies
+
+However, the cluster is not ready to be used at this stage, the nodes are not ready:
+
+{{< codeFromInline lang="bash" >}}
+kubectl get nodes -A
+NAME                 STATUS     ROLES    AGE   VERSION
+kind-control-plane   NotReady   master   94s   v1.17.2
+kind-worker          NotReady   <none>   52s   v1.17.2
+{{< /codeFromInline >}}
+
+and there is no Pods:
+
+{{< codeFromInline lang="bash" >}}
+kubectl get pods -A
+No resources found.
+{{< /codeFromInline >}}
+
+We need to grant the controller access to the policy so it can create the system pods, like CoreDNS, kube-scheduler, ...
+
+Deploy [Authorization Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies).
```

Comment:
might be a good idea to indicate that this example policy is a non-restrictive one.